modules/vba_documents/app/workers/vba_documents/slack_notifier.rb

```diff
@@ -42,55 +41,55 @@ def fetch_settings
 
     def daily_notification
       hour = Time.now.utc.hour + Time.zone_offset('EST') / (60 * 60)
+      results = ''
+
       if hour.eql?(@daily_notification_hour)
-        text = "Daily Status (worst offenders over past week):\n"
         statuses = UploadSubmission::IN_FLIGHT_STATUSES + ['uploaded'] - ['success']
         statuses.each do |status|
           model = UploadSubmission.aged_processing(0, :days, status).where('created_at > ?', 7.days.ago).first
           next unless model
 
           start_time = model.metadata['status'][status]['start']
           duration = distance_of_time_in_words(Time.now.to_i - start_time)
-          text += "\tStatus \'#{status}\' for #{duration}\n"
+          results += "\n\tStatus \'#{status}\' for #{duration}"
```

Comment:
Maybe it used to, for an older version of Ruby? But I also know that this alert has been working for a long time, and it's been modifying the string even with `# frozen_string_literal: true` in place since at least 2021.

I should mention that I felt very tempted to rewrite the modification of the string in favor of constructing the string once outside the loop for all of the alerts, but then drew the line of refactoring here. I can go further with that refactoring if you think it's a good idea. My main reason for stopping was that we'd already added a good amount of scope to the current ticket, and it was starting to feel like a slippery slope.

Reply:
Hmm, all the documentation that I could find explained that `# frozen_string_literal: true` at the top of a file _should_ automatically freeze all strings in the file, but it doesn't seem to be doing so here.